Source/Core/VideoBackends/OGL/BoundingBox.cpp

```diff
@@ -2,69 +2,167 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
+#include <algorithm>
+#include <array>
 #include <cstring>
 
 #include "Common/GL/GLUtil.h"
 
 #include "VideoBackends/OGL/BoundingBox.h"
+#include "VideoBackends/OGL/FramebufferManager.h"
 
 #include "VideoCommon/DriverDetails.h"
 #include "VideoCommon/VideoConfig.h"
 
 static GLuint s_bbox_buffer_id;
+static GLuint s_pbo;
+
+static std::array<int, 4> s_stencil_bounds{{0, 0, 0, 0}};
```

Comment:
Is there initializer list here required? AFAIK static variables should auto-initialize to zero.

Also, since they are static, you may want to clear them at backend shutdown if this is not already done, so the state is consistent across multiple runs.